content/2017.03.building-projects-as-a-distributed-team.rst

```diff
@@ -0,0 +1,178 @@
+but you will realize that if you don't do it, you won't remember on
+what you'd agree and people of your team will start asking questions
+about why you did it like that. If the expectation change along the
+way we should notice it and take decision to update the document.
+
+
+Answering complex questions
+---------------------------
+
+Some features of the MVP might not be trivial. Some decision are hard
+to make between two solutions.
+
+You will have to carefully explain pros and cons and why you choose to
+do it like that. If it is written it is easy to explain people who
+forgot about it why you did it like that.
+
+We have a bunch of examples of hard problems:
+
+- How do you handle permissions in Kinto?
+- How do we support multiple database backends?
+- How do we handle ``content-editable`` divs with React or Elm?
+- How do you implement an OAuth flow from a web extension or a Mobile app?
+- How do you handle pluggable notifications in Kinto?
+- How do we configure email notifications in Kinto?
+- How do we let people attach files to their records in Kinto?
+- How do we get Firefox Account keys derived from the password from
+  an OAuth flow when we never get our hands on the password?
+
+Each problem will have to be addressed by someone and presented to the
+team members so that they can comment on it. Often the way to go is
+obvious — sometimes there is only one good solution — but sometimes
+the good solution is not obvious.
+
+When a complex problem arises, what we want is to learn from it and to
+share what we've learned. A good way of doing so is to write a
+blueprint article exposing pros and cons for the problem and
+explaining which solution will be chosen, why and how we plan on
+implementing it.
+
+
+Conclusion
+==========
+
+Document everything you plan on doing and explain why you plan on
+doing it like that.
+
+- It will be easier for your teamates to follow what you are doing.
+- It will be easier for them to work with you asynchroneously.
+- It will help to show that you are making progress.
+- It will help one to understand how to use your software.
+- It will allow people from your community to give feedback and be
+  implicated in the process.
+- It will also help people to present new ideas and start a discussion
+  about them.
+
+It will help you to handle software as a team rather than as an
+individual, because after each consensus, it is not your solution
+anymore but the team decision to do it like that.
+
+Document your roadmap and product iterations changelog.
+
+If you start coding without having a blueprint, it probably means you
+are wasting your time because you are running alone, building
+something without asking for feedback.
+
+You are not documenting what you are learning which means your
+coworkers won't be able to cach up to help you . You might end-up
+rewritting everything after taking feedback because you missed
+something important.
```

Comment:
I think this paragraph should be joined with the previous one. `catch up`. Superfluous space after `help you`. `end up` (no hyphen). `rewriting` -- the doubled consonant before `ing` would mean that the stem was `rewrit` instead of `rewrite`. `getting feedback`.